src/components/Sidebar/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SidebarToggle from '../SidebarToggle';
+import SidebarItem from '../SidebarItem';
+
+import './Sidebar.scss';
+
+function Sidebar(props) {
+  const {
+    sidebarEnabled, sidebarExpanded, sidebarData, toggleSidebar, itemsToShow,
+  } = props;
+
+  if (!sidebarEnabled) {
+    return null;
+  }
+
+  return sidebarExpanded ? (
+    <div id="sidebar">
+      <SidebarToggle expanded={sidebarExpanded} toggleSidebar={toggleSidebar} />
+      <div className="sidebar-items">
+        {
+          sidebarData.map(item => (
```

Comment:
Note to reviewers: Below data can be used to see sidebar in action. Just use the `SIDEBAR_DATA` in place of `sidebarData`

```python


const SIDEBAR_DATA = [
  {
    category: 'Catalogs',
    showAllButton: true,
    action: {
      type: 'BTN_LINK',
      title: 'Create New Catalog',
      href: 'https://reactjs.org/',
      classes: 'fa fa-plus',
    },
    items: [
      {
        type: 'TXT_LINK',
        title: 'JS Catalog',
        href: 'https://js.org/',
      }, {
        type: 'TXT_LINK',
        title: 'PY Catalog',
        href: 'https://www.python.org/',
      }, {
        type: 'TXT_LINK',
        title: 'ReactJS Catalog',
        href: 'https://reactjs.org/',
      }, {
        type: 'TXT_LINK',
        title: 'GO Catalog',
        href: 'https://www.python.org/',
      }, {
        type: 'TXT_LINK',
        title: 'PHP Catalog',
        href: 'https://js.org/',
      }, {
        type: 'TXT_LINK',
        title: 'Google Catalog',
        href: 'https://www.python.org/',
      }, {
        type: 'TXT_LINK',
        title: 'Rust Catalog',
        href: 'https://js.org/',
      }, {
        type: 'TXT_LINK',
        title: 'Java Catalog',
        href: 'https://www.python.org/',
      }, {
        type: 'TXT_LINK',
        title: 'Redux Catalog',
        href: 'https://www.python.org/',
      },
    ],
  }, {
    category: 'Learner Management',
    showAllButton: false,
    items: [
      {
        type: 'TXT_LINK',
        title: 'Peoeple & Groups',
        href: 'https://edx.org/',
      }, {
        type: 'TXT_LINK',
        title: 'Account Finances',
        href: 'https://edx.org/',
      }, {
        type: 'TXT_LINK',
        title: 'Analytics',
        href: 'https://edx.org/',
      }, {
        type: 'TXT_LINK',
        title: 'Admin Settings',
        href: 'https://edx.org/',
      }, {
        type: 'TXT_LINK',
        title: 'Help & FAQ',
        href: 'https://edx.org/',
      },
    ],
  },
];
```